modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/roundstartslime.dm

```diff
@@ -62,6 +62,19 @@
 	var/slime_restricted = TRUE
 	///Is the person using this ability oversized?
 	var/oversized_user = FALSE
+	///List containing all of the avalible parts
+	var/static/list/avalible_choices
+
+/datum/action/innate/alter_form/New(Target)
+	. = ..()
+	var/list/parts_lists = GLOB.sprite_accessories.Copy()
+	for(var/parts_list in parts_lists)
+		for(var/parts as anything in parts_lists[parts_list])
+			var/datum/sprite_accessory/part = parts_lists[parts_list][parts]
+			if(part.locked)
+				parts_lists[parts_list] -= parts
+
+	avalible_choices = parts_lists.Copy()
```

Comment:
You should only run the whole "parts_list" part if `available_choices` is null, otherwise it defeats the point in having it as a static thing.

Reply:
Also it's `available` not `avalible` :^)